### PR TITLE
KC-106 Fix bugs in frontend

### PR DESCRIPTION
**Pull request by hottek**

### CODE REVIEW

File: dashboard/src/app/dialog/dialog.component.ts [hunk 1]
```diff
@@ -24,8 +24,7 @@ export class DialogComponent implements OnInit {
   }
 
   generatePassword() {
-    this.password = 'asdf';
-    // this.generatedPassword = passwordGenerator.genPW(); TODO: fix bug in pwgen.js, see console for error message
+    this.password = passwordGenerator.genPW();
```
Comment (zkdev) [inline dashboard/src/app/dialog/dialog.component.ts:27]: Should we check whether the password is really secure? (16 chars, lower, upper, number, special chars)

Comment (hottek) [reply dashboard/src/app/dialog/dialog.component.ts:27]: have a look at [this](https://github.com/keycloud/keycloud/blob/master/dashboard/src/app/util/pwgen.js). You can find the generation there.
  The generated password is checked against the [haveibeenpawned](https://haveibeenpwned.com/)-api.

Comment (hottek) [reply dashboard/src/app/dialog/dialog.component.ts:27]: [This line and the two following](https://github.com/keycloud/keycloud/blob/50602d2dc11b79a3d1b05b71c64473c645681bc6/dashboard/src/app/util/pwgen.js#L31) specify which chars are used during password generation.

Comment (zkdev) [reply dashboard/src/app/dialog/dialog.component.ts:27]: good one.

**Comment by zkdev** [comment]
@hottek please remember to delete branches once merged

**Pull request merged**